packages/calypso-e2e/src/browser-manager.ts

```diff
@@ -84,6 +84,7 @@ export async function startBrowser( browserType: BrowserType ): Promise< Browser
 	}
 
 	browser = await browserType.launch( {
+		channel: 'chrome',
```

Comment:
I wanted to avoid this change, but it appears this is necessary if we are to move off Playwright 1.14.

In TeamCity, using the bundled Chromium browser on Playwright version 1.14+ leads to either:

Outright failure to run any tests 
![image](https://user-images.githubusercontent.com/6549265/144528861-133fc1c0-3b06-4cc3-9053-7d42fca20696.png)

or

Handful of tests failing persistently
![image](https://user-images.githubusercontent.com/6549265/144528914-9aedc75e-8aae-4ede-ab5a-0f4c1fe8ff45.png)

It is not known for sure why this is the case, as running this locally with Chromium (headless and headed) does not reproduce the issue. Even more oddly, these failures do not result in trace files or screenshots/videos being uploaded as artifact, making debugging quite difficult.

It is possible to use the bundled Chromium version to obtain a passing run, but that would mean running this in headed mode, requiring the use of `xvfb-run`.

Reply:
Another thought occurred to me: could this possibly due to our custom docker image? 

For reference, [this is the docker image](https://github.com/microsoft/playwright/blob/master/utils/docker/Dockerfile.bionic) supplied by the Playwright team.

Reply:
I tested it locally using our latest docker image + Xquartz and running `yarn jest ./specs/specs-playwright/wp-gutenboarding__happy-path.ts` and works for me both headless and headed mode